platform/local_country_file_utils.hpp

```diff
@@ -42,7 +46,11 @@ void FindAllLocalMapsInDirectoryAndCleanup(string const & directory, int64_t ver
 //
 // Also, this method performs cleanup described in a comment for
 // CleanupMapsDirectory().
-void FindAllLocalMapsAndCleanup(int64_t latestVersion, vector<LocalCountryFile> & localFiles);
+// Note. The the maps are looked for writable dir/|dataDir|/|version| directory.
+// If |dataDir| is empty (or is not set) the function looks for maps in writable dir/|dataDir|/.
+void FindAllLocalMapsAndCleanup(vector<LocalCountryFile> & localFiles, int64_t latestVersion);
```

Comment:
Why |localFiles| is the first parameter? All output parameters should be last, right?